src/App.jsx

```diff
@@ -14,70 +15,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const empty = 'No goods selected';
+  const [value, setValue] = useState('Jam');
+  const reset = () => setValue(empty);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {(value !== empty)
```

Comment:
weird condition, you can do like below:

`{value ? `${value} is selected` : 'No goods selected'}`

Reply:
we must add ClearButton if selectorGood don't selected. if i doing it how you tell we have that button always

Reply:
sorry. If selectorGood is not empty